android/build.gradle

```diff
@@ -737,7 +868,7 @@ afterEvaluate {
 
 if (CLIENT_SIDE_BUILD) {
```

Comment:
Are `CLIENT_SIDE_BUILD` and `BUILD_FROM_SOURCE` somehow related? If yes, can we use only `BUILD_FROM_SOURCE`?